### DISCUSSION TITLE
default port 8000 can be change?

### DISCUSSION THREAD

**Discussion by rotikia** [mpromonet/webrtc-streamer#590] [General]
my site port 8000 is using by others app.
I try to find in all the downloaded file and cannot find any config which related to this number of 8000.
Can suggest me a method to change the port 8000 to others port number? 
thanks

**Comment by mpromonet** [comment]
Hi,

You can change the listening port adding to command line '-H <port>' or setting the environnent variable PORT.

Best Regards,
Michel.
